python/test/k8s-test-pydevd-python310.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: python310pod
+  labels:
+    app: hello
+    protocol: pydevd
+    runtime: python310
+spec:
+  containers:
+  - name: python310app
+    image: python310app
+    command: ["/dbg/python/launcher", "--mode", "pydevd", "--port", "12345", "--"]
+    args: ["python", "-m", "flask", "run", "--host=0.0.0.0"]
+    ports:
+    - containerPort: 5000
+    - containerPort: 12345
+      name: pydevd
+    env:
+    - name: WRAPPER_VERBOSE
+      value: debug
+    readinessProbe:
+      httpGet:
+        path: /
+        port: 5000
+    volumeMounts:
+    - mountPath: /dbg
+      name: python-debugging-support
+  initContainers:
```

Comment:
won't skaffold add these `initContainers` at runtime? Do these need to be added here anyway?

Reply:
These tests aren't launched using `skaffold debug` — it's mimicking what `skaffold debug` will do.  That's necessary as ideally we'll be testing all major runtime debugging options (e.g., pydevd and debugpy in this Python case) whereas `skaffold debug` will only choose  a single option.